test/Microsoft.ML.Tests/Transformers/NormalizerTests.cs

```diff
@@ -383,6 +387,45 @@ public void NormalizerParametersMultiColumnApi()
             Assert.Equal(4, doubleBinModelVector.Density.Length);
             Assert.Empty(doubleBinModelVector.Offset);
 
+            // Robust scaler
+            var robustScalerEstimator = context.Transforms.NormalizeRobustScaling(
+                                new[] {new InputOutputColumnPair("float1rbs", "float1"), new InputOutputColumnPair("float4rbs", "float4"),
```

Comment:
The normalizer API haven't been tested thoroughly through Onnx I think. Can you please add a test in OnnxConversionTest.cs for onnx export for this and other Normalizer apis ?

Reply:
I added in the ONNX export tests. We dont' support export for anything where `useCdf` is true, and neither of the binning normalizers support it, but the rest now have tests.